src/pare-json.js

```diff
@@ -1,10 +1,16 @@
-const yargs = require('yargs/yargs');
+const minimistParser = require('minimist');
 const convertor = require('./convertor');
 
 const options = require('./option');
 
 module.exports = exports.default = function (data) {
-	const argv = yargs(data).argv
+	if (typeof data === 'string' || data instanceof String) {
+		data = data
+			.replace(/\\\n/g, ' ')
+			.match(/"([^"\\]*(?:\\.[^"\\]*)*)"|'([^'\\]*(?:\\.[^'\\]*)*)'|[^\s]+/g)
+			.map((s) => s.replace(/('|")/g, ''))
+	}
```

Comment:
Can you add a unit test for that? I don't know how it affects the main feature, but if you add a test case, it can tell me how it works!

I want to make sure it never crashes our tool.

Reply:
```js
data = data
			.replace(/\\\n/g, ' ')
			.match(/"([^"\\]*(?:\\.[^"\\]*)*)"|'([^'\\]*(?:\\.[^'\\]*)*)'|[^\s]+/g)
			.map((s) => s.replace(/('|")/g, ''))
```

You can move it to a new file (such as utils.js, regex.js, etc.) that will be easy to implement test cases.